docs/production/containers.rst

```diff
@@ -130,27 +164,39 @@ It will probably be neccesary to add the user to the Docker group (this will tak
 
 For the next command you will probably need to log out and back in to load the group change.
 
+.. _docker_installation_windows:
 
-.. _podman_installation:
+Docker Installation - WINDOWS
+-----------------------------
+.. warning::
+	This tutorial covers the installation on Windows 10 Build 2004 or later and should be compatible
+	with Windows 11 as well.
 
-Podman Installation
--------------------
 
-.. note::
-	If you already installed docker you can skip this part
+In this tutorial we will be working with Docker Desktop, which is free for non-commercial usage. 
+If you plan on deploying the bot in commercial environment, consider using Rancher Desktop.
 
-The first step is installing the podman, podman-docker and docker-compose:
+As it's recommended to use Docker Desktop with WSL2 backend (instead of HyperV), 
+this tutorial will cover the WSL2 installation as well.
 
-.. code-block:: bash
+We recommend creating ``.wslconfig`` file in your ``userprofile`` folder to enable ``sparseVhd`` option.
+This will automatically shrink virtual hard drives of the WSL2. This function is supported since September 2023 update.
+The file should contain this section:
 
-	sudo apt install podman podman-docker docker-compose
+.. codeblock:: 
 
-Start the Podman system service:
+	[experimental]
+	sparseVhd=true
 
-.. code-block:: bash
+If you don't have WSL2 installend, you must run following command in ``cmd`` as Administrator.
```

Comment:
Typo in installed.
I'd use need to or have to instead of must.

Reply:
Changed